cylc/rose/entry_points.py

```diff
@@ -134,6 +134,9 @@ def rose_fileinstall(dir_=None, opts=None, dest_root=None):
         finally:
             os.chdir(startpoint)
 
+    from cylc.rose.utilities import dump_rose_log
```

Comment:
Have already imported from `cylc.rose.utilities` at the top of the file so no performance improvement to doing it later on.

Reply:
And because I'm having that sort of day I alphabetized the imports too.

Reply:
Still needs to be moved to the top?

Reply:
Yes. I've no idea why I've missed that.